tests/pr_test.go

```diff
@@ -69,6 +69,7 @@ func setupOptions(t *testing.T, prefix string, exampleDir string) *testhelper.Te
 		"prefix":         options.Prefix,
 		"resource_group": resourceGroup,
 		"resource_tags":  options.Tags,
+		"plan":           "essentials",
```

Comment:
Can you remove this line? This will result in the [error](https://github.com/terraform-ibm-modules/terraform-ibm-watsonx-orchestrate/actions/runs/15067149980/job/42354705213#step:7:593) as `plan` is not declared in the basic example and makes use of the default plan provided in the root module.

It is fine to change the plan to `essentials` at other places.